lab-yunjoo/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab-yunjoo",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "bcrypt": "^0.8.6",
```

Comment:
Bcrypt, body-parser, express, http-errors, mongoose, and morgan, and debug belong in dependencies. They are not devDependencies.